src/pyfaf/actions/save_reports.py

```diff
@@ -306,36 +308,33 @@ def handle_term(_, __):
     def _save_attachments(self, db):
         self.log_info("Saving attachments")
 
-        attachment_filenames = os.listdir(self.dir_attach_incoming)
+        with os.scandir(self.dir_attach_incoming) as iterator:
+            for i, entry in enumerate(iterator, start=1):
+                self.log_info("[%d / ??] Processing file '%s'" % (i, entry.path))
```

Comment:
I am not a fan of "??".

We don't recieve that many attachments (from logs its ~500 per week).

So IMO just drop the enumeration and replace the message with:
```
Processing attachment file "b14b14b14"
```

Reply:
Good, that makes two non-fans.

Reply:
Changed.